fuzzers/libfuzzer_stb_image/src/main.rs

```diff
@@ -10,17 +10,17 @@ use libafl::{
         QueueCorpusScheduler,
     },
     events::{setup_restarting_mgr_std, EventConfig},
-    executors::{inprocess::InProcessExecutor, ExitKind, ShadowExecutor},
+    executors::{inprocess::InProcessExecutor, ExitKind, ShadowExecutor, timeout::TimeoutExecutor},
     feedback_or,
     feedbacks::{CrashFeedback, MapFeedbackState, MaxMapFeedback, TimeFeedback},
     fuzzer::{Fuzzer, StdFuzzer},
     inputs::{BytesInput, HasTargetBytes},
-    monitors::MultiMonitor,
     mutators::scheduled::{havoc_mutations, StdScheduledMutator},
     mutators::token_mutations::I2SRandReplace,
     observers::{StdMapObserver, TimeObserver},
     stages::{ShadowTracingStage, StdMutationalStage},
     state::{HasCorpus, StdState},
+    stats::MultiStats,
```

Comment:
Something went wrong here :)
We renamed stats to monitors

Reply:
I'll revert changes on libfuzzer_stb_image/src/main.rs and harness.cc